### ISSUE TITLE
build.yaml in root dir needs to be removed.  

### ISSUE THREAD

**Issue by tobytraylor** [brianegan/flutter_architecture_samples#81]
You have a build.yaml checked into the root dir.  It will break with the current versions of flutter and built_value, built_value_generator.

**Comment by brianegan** [comment]
Hey there -- sorry, I don't quite see that file either on Github or on my local filesystem. The only `build.yaml` I can spot is inside the `built_redux` example.

Could you point me to a screenshot or similar of what you're seeing?

**Comment by tobytraylor** [comment]
Right.  Sorry that was very inaccurate of me.  It's the build.yaml inside the built_redux example.
https://github.com/brianegan/flutter_architecture_samples/blob/master/example/built_redux/build.yaml.

It fails because both built_redux and built_value_generator have been updated to the latest build and source_gen versions so the build file being generated by running build_runner is different from the one you have checked in.  Unfortunately though it generates the right build file, somehow (and i don't know how) the runner picks up your version from the root file system for the "built_value_generator|built_value:" key and keeps it from generating the files.

**Comment by brianegan** [comment]
Ah dang -- more builds more problems :P Thanks for the info!

**Comment by brianegan** [comment]
Sure thing! Glad it was a helpful starting point :)

**Comment by mmcc007** [comment]
There's a  temporary check for this during the build:
https://github.com/brianegan/flutter_architecture_samples/blob/master/scripts/runTests.sh#L32
and
https://github.com/brianegan/flutter_architecture_samples/blob/master/scripts/ci.sh#L36

The existing generated files still work. But yes, should be fixed.

BTW: looks like I have a fix for current build... hopefully will have a PR soon.
